model/src/main/java/uk/gov/hmcts/payment/api/util/PaymentMethodType.java

```diff
@@ -3,7 +3,10 @@
 public enum PaymentMethodType {
 
     CARD("card"),
-    PBA("payment by account");
+    PBA("payment by account"),
+    CASH("cash"),
+    CHEQUE("cheque"),
```

Comment:
We've a similar enum Method.java . Do you mind to remove this and update references with  PaymentMethodType if possible? 
https://github.com/hmcts/ccpay-payment-app/blob/master/api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/util/Method.java

Reply:
The existing /payments implementation uses PaymentMethodType enum

paymentMethodType.map(value -> PaymentMethodType.valueOf(value.toUpperCase()).getType()).orElse(null);

So I introduced the new types here.

Reply:
Yes, that's right.  But I'm asking a cleanup - delete duplicate enum (Method.java) if possible.

Reply:
cheers, will do it